pkg/kubelet/apis/resourcemetrics/v1alpha1/config.go

```diff
@@ -16,110 +16,8 @@ limitations under the License.
 
 package v1alpha1
 
-import (
-	"time"
-
-	"k8s.io/component-base/metrics"
-	summary "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
-	"k8s.io/kubernetes/pkg/kubelet/server/stats"
-)
-
-// This file contains a series of deprecated metrics which we emit them by endpoint `/metrics/resource/v1alpha1`.
-// These metrics have been adapted to new endpoint `/metrics/resource` as well as new `Desc`s.
-// In general, we don't need to maintain these deprecated metrics any more.
-// TODO(RainbowMango): Remove this file in release 1.20.0+.
+// TODO(RainbowMango): We don't need to maintain this package anymore.
+//  This package will be remove in release 1.20.0+. More details please refer to https://github.com/kubernetes/kubernetes/pull/86282.
 
 // Version is the string representation of the version of this configuration
 const Version = "v1alpha1"
```

Comment:
Out of curiosity, why are we keeping this file with just the `Version` constant defined?

Do we not want to keep the version definition with the actual `NewDesc` descriptions `pkg/kubelet/server/stats/prometheus_resource_metrics.go`?

Reply:
It's no big deal in my opinion. Given the constant also be referred by other components, such as [E2E](https://github.com/kubernetes/kubernetes/blob/6e35a13815adf221bc2b65eadcdb326de2283521/test/e2e_node/resource_metrics_test.go#L114), so I'd prefer to clean it at release 1.20.